shared/message/support/optimisation/NSGA2OptimiserMessages.proto

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Might be good to rename this `NSGA2Optimiser` since we already know it's a message?